src/main/java/edu/cornell/kfs/module/cam/rest/resource/CuCapAssetInventoryApiResource.java

```diff
@@ -122,11 +165,18 @@ private Properties getAssetProperties(Asset asset) {
         return assetProperties;
     }
 
-    private Properties getBuildingProperties(Building buillding) {
+    private Properties getBuildingProperties(Building building) {
+        Properties buildingProperties = new Properties();
+        safelyAddProperty(buildingProperties, CuCamsConstants.CapAssetApi.CAMPUS_CODE, building.getCampusCode());
+        safelyAddProperty(buildingProperties, CuCamsConstants.CapAssetApi.BUILDING_CODE, building.getBuildingCode());
+        safelyAddProperty(buildingProperties, CuCamsConstants.CapAssetApi.BUILDING_NAME, building.getBuildingName());
+        return buildingProperties;
+    }
+
+    private Properties buildAssetConditionProperties(AssetCondition condition) {
         Properties buildingProperties = new Properties();
```

Comment:
The naming is a little confusing here. Should the "buildingProperties" variable be renamed to something like "assetConditionProperties" instead, so that it's more in line with what the method seems to be returning?